lib/bin/rake

```diff
@@ -0,0 +1,22 @@
+#!/Users/brandonfish/Documents/jruby/lib/bin/truffleruby
```

Comment:
Everything in `lib/bin` should be a Ruby script, designed to run with `-S`. `bin` should shell scripts that perhaps run something from `lib/bin` with `-S`. The reason for this is that if `bin` files are Ruby files with shebangs, which Ruby would they run? They rely on the Ruby you want being `env ruby`.